ios-base/Extensions/FontExtension.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FontExtension.swift
+//  ios-base
+//
+//  Created by Germán Stábile on 2/28/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+import RSFontSizes
+
+extension UIFont {
+  var bold: UIFont { return withWeight(.bold) }
+  var semibold: UIFont { return withWeight(.semibold) }
+  var light: UIFont { return withWeight(.ultraLight) }
```

Comment:
Note: we can add as many types as we need. 
Usage sample: 
`titleLabel.font = UIFont.font(withName: "ComicSans", size: 22).bold`

Reply:
Im pretty sure @beluu004 is not going to like that example.

Reply:
Lol, example inspired by the Ghiorse bros

Reply:
🙄 really? @germanStabile @glm4

Reply:
@germanStabile Is there any way to wrap this extension around `RSFontSizes.Font`? One of the reasons to use RSFontSizes is not to repeat `UIFont.font(withName: "ComicSans",..` every time. I think it would be great to do something like `Font.comicSans.bold` or similar.
@pMalvasio @mato2593 @CamilaMoscatelli thoughts?

Reply:
As long as `Font.comicSans` returns a Font you can definitely do that